py3status/modules/systemd.py

```diff
@@ -0,0 +1,88 @@
+# -*- coding: utf-8 -*-
+"""
+Check systemd unit status.
+
+Check the status of a systemd unit.
+
+Configuration parameters:
+    cache_timeout: How often we refresh this module in seconds (default 5)
+    format: Format for module output (default "{unit}: {status}")
+    unit: Name of the unit (default None)
```

Comment:
you need to update the default `(default 'dbus.service')` as currently the tests are failing.